opm/parser/eclipse/EclipseState/Tables/Tables.cpp

```diff
@@ -1085,6 +1085,22 @@ struct flat_props< PVTWRecord, 0 > {
     }
 };
 
+constexpr const char* pvcdo_err[] = {
```

Comment:
Ok.

Please note that these keywords having error messages (defaulted value where it is illegal) is the *exception*. I valued making the common case simpler more. PVCDO is also special because *none* of the fields can be defaulted, in which case it was easily worked around. Typically only the first item cannot be defaulted, if any at all, in which case you only implement that.